src/store/socket.js

```diff
@@ -684,6 +691,9 @@ class LiveSession {
   setIsNight() {
     if (this._isSpectator) return;
     this._send("isNight", this._store.state.grimoire.isNight);
+    if (this._store.state.grimoire.isNight) {
+      this._store.commit("players/setOnBlock", -1);
```

Comment:
This doesn't (visually) remove the "marked" flag for the player in the ST town square, only after interacting on the player element does it actually update the mark status. 
Also, the socket.js shouldn't contain "game logic", so this should happen on the Menu where Night is toggled rather than here.

Reply:
Can do this, but I felt it was less tidy because night is triggered from multiple code paths (menu, hotkey).

Reply:
Weird, I can't reproduce it anymore now either. Hm, maybe it depends on something else.
Anyhow, I'd move the logic to its own method in Menu.vue and then have the hotkey call in App.vue just access the menu function, like when you join a session through a hotkey.